app/javascript/components/time_table/project_reports/project_reports.js

```diff
@@ -50,66 +50,85 @@ export default class ProjectReports extends React.Component {
       });
   }
 
+  renderReportState(state) {
+    let iconClass;
+    switch (state) {
+      case 'done':
+        iconClass = 'fa-check';
+        break;
+      case 'editing':
+        iconClass = 'fa-pencil';
+        break;
+      default:
+        iconClass = 'fa-info-circle';
+    }
```

Comment:
we could simplify this into something like let iconClass = ({'done' : 'fa-check', 'editing': 'fa-pencil'})[state] || 'fa-info-circle' So fetching object value by key with default